README.md

```diff
@@ -12,13 +12,13 @@ The contents of this code of conduct apply to our interactions in various areas
 
 ## Values
 
-We want our team to be a fun, productive, and safe space for all members. In addition to core values of respect and honesty, there are several ways in which our product team distinguishes itself.
+We want our team to be fun, productive, and safe for all members. In addition to the Social Tables [core values](https://www.socialtables.com/company/about/#values), there are several ways in which our Engineering team distinguishes itself.
```

Comment:
All good points. I do need to update https://github.com/socialtables/convo/issues/83 to tell everyone that we'll need to wait for the company initiative to rework the Core Values so we can align this with those.

I'm pretty open to being talked into making this Engineering + Product. I fear that making it company-wide would undermine the value to the Engineering team. This is in part due to the fact that the issues we're talking about are particularly acute in engineering culture and present barriers to achieving diversity on engineering teams. When our Engineering team says this is the culture we stand for, it makes a strong statement. I worry that if the CoC were a company-wide initiative, the Engineering team loses that -- there's no way to tell whether or not we really care. The same can be said of expanding it to Product, but to a much lesser extent (I think).